btcutil/psbt/psbt.go

```diff
@@ -214,22 +214,12 @@ func NewFromRawBytes(r io.Reader, b64 bool) (*Packet, error) {
 		return nil, err
 	}
 	msgTx := wire.NewMsgTx(2)
-	err = msgTx.Deserialize(bytes.NewReader(value))
+
+	// BIP-0174 states: "The transaction must be in the old serialization
+	// format (without witnesses)."
+	err = msgTx.DeserializeNoWitness(bytes.NewReader(value))
```

Comment:
Should we continue to try to decode the witness version and fall back to non-witness? Since it's possible there's old software out there encoding wit the witness version, we'd potentially break compatibility across lnd version for even btcutil versions with this change as is. 

It seems the most important thing here is for us to _encode_ using the new witness version, AFAICT. So if we just serialize w/o the witness, then things are compatible across the prior versions.

Reply:
I can add the fallback. Though I don't think this change will create a compatibility issue. A PSBT packet with an unsigned TX that has a witness set cannot be serialized, the library will complain. And if you call `msgTx.Serialize()` with a transaction that has no witness, the old format will be used. Therefore the change to the serialization is purely to be more explicit about it. The actual important change is that we always _read_ the TX expecting it to be in the non-witness format.